packages/atom/package.json

```
@@ -2,21 +2,22 @@
   "name": "@effection/atom",
   "version": "2.0.0-beta.7",
   "description": "State atom implementation for effection",
-  "main": "dist/cjs/index.js",
-  "types": "dist/esm/index.d.ts",
+  "main": "dist-cjs/index.js",
+  "types": "dist-esm/index.d.ts",
+  "typeDocEntry": "src/index.ts",
   "homepage": "https://github.com/thefrontside/effection",
   "repository": {
     "type": "git",
     "url": "https://github.com/thefrontside/effection.git",
     "directory": "packages/atom"
   },
-  "typeDocEntry": "src/index.ts",
   "author": "Frontside Engineering <engineering@frontside.io>",
   "license": "MIT",
   "files": [
     "README.md",
     "CHANGELOG.md",
-    "dist/**/*",
+    "dist-esm/**/*",
+    "dist-cjs/**/*",
```

We could make this a little more DRY
```suggestion
    "dist-*/**/*",
```

---

oh that is true :+1: